webapp-client/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "parser": "babel-eslint",
+  "env": {
+    "es6": true,
+    "node": true,
+    "browser": true
+  },
+  "rules": {
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "react/destructuring-assignment": "warn",
+    "spaced-comment": ["error", "always"]
+  },
+  "settings": {
+    "ecmascript": 6,
+    "jsx": true
+  },
+  "plugins": ["react"],
+  "extends": ["airbnb", "eslint:recommended", "plugin:react/recommended"]
```

Comment:
If you're using these like the airbnb config, do you have to npm install it too?

Also consider using https://github.com/google/eslint-config-google/blob/master/index.js too?

Reply:
Oops I copy pasted the wrong link: https://github.com/google/eslint-config-google